question_2.py

```diff
@@ -0,0 +1,23 @@
+class Person:
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+    
+    def display(self) -> None:
+        print(f"[INFO] <Name>: {self.name}", end=" ")
```

Comment:
doc string is missing

Reply:
I thought the display function was pretty self explanatory, anyways I have added a doc string.

Reply:
Always follow best practices while coding.